sklearn/neural_network/_stochastic_optimizers.py

```diff
@@ -73,6 +73,9 @@ def trigger_stopping(self, msg, verbose):
 class SGDOptimizer(BaseOptimizer):
     """Stochastic gradient descent optimizer with momentum
 
+    For example usage and comparison with Adam optimizer, see
+    :ref:`sphx_glr_auto_examples_neural_networks_plot_mlp_training_curves.py`
```

Comment:
the link to this example should also go under the `solver` attribute on `MLPClassifier` and `MLPRegressor`